README.md

```diff
@@ -60,6 +61,7 @@ This list contains tools, books, articles, blogs, courses and everything related
 ### Translation software
 - [OmegaT](https://omegat.org/) - Free translation memory / CAT software that works on Windows, macOS and Linux.
 - [Argos Translate](https://github.com/argosopentech/argos-translate) - Completely open source offline translation app written in Python and based on OpenNMT. It can also be used as a Python library.
+- [Fink](https://inlang.com/m/tdozzpar) - Translation editor / CAT software with no-code setup and repository-based operation.
```

Comment:
Provided link `https://inlang.com/m/tdozzpar` redirect to `https://inlang.com/m/tdozzpar/app-inlang-editor`. Similar below (line 98), from `https://inlang.com/m/gerre34r` redirected to `https://inlang.com/m/gerre34r/library-inlang-paraglidejs`. `https://inlang.com/m/` gives `main site`. Shouldn't we use direct links there?

Reply:
For example: what if we will have `https://inlang.com/m/tdozzpar/**item2**` and similar? How can we be sure that provided `https://inlang.com/m/tdozzpar` will still redirect correctly end-user to the mentioned `https://inlang.com/m/tdozzpar/app-inlang-editor`? Sorry for asking, and thank you in advance for the answer.

Reply:
We could also use the full link. I think this follows the same system as amazon products. The id stays the same and if inlang decides to rebrand or change SEO names ... it will still redirect to the same page. 

So I  think, id is less fluent than the name.

But it's up to you :) Let me know how you want it to be.

Reply:
Thank you for the explanation! All is fine 🚀